pages/implementation/start-using-inspector-with-avo-functions.mdx

```diff
@@ -1,51 +1,57 @@
 ---
 layout: 'guide'
-title: 'Using Inspector with Avo Functions'
+title: 'Getting started to use Inspector alongside Avo Functions'
 ---
 
+import Link from '../../components/Link';
+
 You need to do minor setup to enable **Avo Functions** reporting to **Avo Inspector**.
 
 Common steps for all platforms:
 
-1. Avo Inspector should be enabled for given source. Open `Inspector` in the side menu in your [Avo workspace](https://www.avo.app/schemas/), then press "Manage Sources" button at the top, choose your source from the grid and wait until you see the `API Key`. Once the `API Key` appears you source is considered Inspector-enabled.
+1. Avo Inspector should be enabled for given source. Open <Link passHref href="/workspace/inspector"><a>the Inspector tab</a></Link> in the side menu in your [Avo workspace](https://www.avo.app/schemas/), then press "Manage Sources" button at the top, choose your source from the grid and wait until you see the `API Key`. Once the `API Key` appears you source is considered Inspector-enabled.
 
 2. Your Avo generated file should use Avo version 63.10.0 or greater. You can find the version in the comment at the top of the generated file. If the version is older than 63.10.0 run `avo pull` again.
 
-
 Platform specific setup:
 
 <details>
 
-<summary><strong>iOS</strong></summary>
-<p/>
-Avo Functions report to Avo Inspector automatically starting with Avo version >= 63.10.0 and Inspector SDK version >= 1.2.0.
+
+<summary>
+  <strong>iOS</strong>
+</summary>
```

Comment:
My main idea here was to limit the shown instructions to a single platform because they are quite similar and it seems to be easy to mix them up accidentally.